saleor/graphql/payment/tests/mutations/test_transaction_request_action.py

```diff
@@ -573,3 +615,13 @@ def test_transaction_request_action_missing_event(
     )
 
     assert mocked_is_active.called
+
+    assert (
+        len(
+            TransactionEvent.objects.filter(
+                transaction=transaction,
+                status=TransactionEventStatus.REQUEST,
+            )
+        )
+        == 0
+    )
```

Comment:
You could use `exists` here:
```suggestion
assert not TransactionEvent.objects.filter(
      transaction=transaction,
      status=TransactionEventStatus.REQUEST,
  ).exists()
```